pkgs/top-level/all-packages.nix

```diff
@@ -964,6 +964,8 @@ in
 
   bfr = callPackage ../tools/misc/bfr { };
 
+  bftools = callPackage ../applications/science/biology/bftools { };
```

Comment:
In principle there is whole section for SCIENCE/BIOLOGY down there:
https://github.com/NixOS/nixpkgs/blob/6bc8a6621296d073974c4537933b53aa029edde5/pkgs/top-level/all-packages.nix#L21415